neural_compressor/adaptor/torch_utils/smooth_quant.py

```diff
@@ -706,6 +706,15 @@ def transform(self, alpha=0.5, folding=False, percentile=99.999, op_types=['Line
             if need_calibration:  ##avoid multiple calibaration during tuning if the only difference is alpha
                 if self.insert_mul:
                     self.self_absorb_layers = self._get_all_layer_names()  # TODO: only support linear now.
+                    # fetch modules with the same input
+                    group_modules = self._trace(op_types, skip_unsupported_layers=False)
```

Comment:
Tracing is not so robust. if tracing has some issues, could we fallback to checking the calibration tensor though it's a little trick